src/templates/videoTemplate.html

```diff
@@ -17,10 +17,11 @@
                                 <li>Open in Invidious</li>
                             </a>
                             <li v-on:click='copy("invidio.us", video.id)' onclick='showVideoOptions(this.parentNode.previousSibling);'>Copy Invidious Link</li>
+                            <li onclick='clickMiniPlayer(this.parentNode.querySelectorAll("a")[0].href)'>Open in Mini Player</li>
```

Comment:
Clicking on this should close the 3 dot menu.

Reply:
Done!